src/website/src/app/documentation/demos/i18n/i18n.demo.ts

```diff
@@ -55,5 +55,8 @@ export class I18nDemo extends ClarityDocComponent {
     },
     { key: 'allColumnsSelected', role: 'Datagrid: screen reader only confirmation that all columns were selected' },
     { key: 'loading', role: 'Display loading text (Default: Loading)' },
+    { key: 'singleSelectionAriaLabel', role: 'Datagrid: AriaLabel for single selection in header' },
+    { key: 'singleActionableAriaLabel', role: 'Datagrid: AriaLabel for Actionable selection in header' },
```

Comment:
nit: `Datagrid: aria label for header single selection header column` and `Datagrid: aria label for row action header column`